src/drawer.js

```diff
@@ -600,6 +600,28 @@ $.Drawer.prototype = {
         }
     },
 
+
+    /**
+     * Turns image smoothing on or off for this viewer. Note: Ignored in some (especially older) browsers that do not support this property.
+     *
+     * @function
+     * @param {Boolean} [imageSmoothingEnabled] - Whether or not the image is
+     * drawn smoothly on the canvas; see imageSmoothingEnabled in
+     * {@link OpenSeadragon.Options} for more explanation.
+     */
+    setImageSmoothingEnabled: function(imageSmoothingEnabled){
+        if ( this.useCanvas ) {
+            this.context.mozImageSmoothingEnabled = imageSmoothingEnabled;
+            this.context.webkitImageSmoothingEnabled = imageSmoothingEnabled;
+            this.context.msImageSmoothingEnabled = imageSmoothingEnabled;
+            this.context.imageSmoothingEnabled = imageSmoothingEnabled;
+
+            this.context.restore();
```

Comment:
Why this `restore`? Seems like it would cause an imbalance without a `save`.

Reply:
Oh, I forgot the save. Included it. I saw this behaviour (save restore) at other functions.